### DISCUSSION TITLE
How can two clusters share one minio storage??

### DISCUSSION THREAD

**Discussion by misteruly** [vmware-tanzu/velero#3246] [Community support Q&A] [answered]
I now have multiple clusters and want to back up to the same minio storage. Is there a good way?

**Comment by carlisia** [comment]
You may run Minio in a Docker container on a machine, and configure the Velero backup location in each cluster to connect to it. This is how to do it:

https://github.com/vmware-tanzu/velero/wiki/Contributing-FAQ#how-do-i-run-minio-locally-on-my-computer-as-a-standalone-as-opposed-to-running-it-on-a-pod.

**Comment by stoneshi-yunify** [comment]
@carlisia Can I use AccessKey and SecretKey instead of cacert for a shared minio? How to do that?

**Comment by misteruly** [comment, marked as answer]
@carlisia   yes,right.But how do you migrate PV data???

**Reply by dsu-igeek** [reply]
In order to migrate PV data between clusters that do not share a storage system, use Restic.  https://velero.io/docs/v1.5/restic/
